hw2/aes_ttable_cache_attack.py

```diff
@@ -82,7 +87,7 @@ def simulate_cache_access(plaintexts, key, start_round, end_round):
             stateround = aes.encrypt_r(plaintexts[j], r, start_round)
             for i in range(NumTables):
                 for l in range(int(BlockSize / NumTables)):
-                    accessed_list[j][i].append( ? )
+                    accessed_list[j][i].append(stateround[(NumTables * l + (i * int(BlockSize / NumTables + 1))) % BlockSize] &0xF0)

```

Comment:
Maybe we can add a function of
```py
def state_byte_index_used_to_access(table_index, column_index):
    return (NumTables * column_index + (table_index * int(BlockSize / NumTables + 1))) % BlockSize
```
?

I'm not sure about the name, but it can be a bit more descriptive this way.